Packet++/header/InfiniBandLayer.h

```diff
@@ -0,0 +1,288 @@
+#pragma once
+
+#include "Layer.h"
+
+/// @file
+
+/**
+ * \namespace pcpp
+ * \brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+	/**
+	 * IBA header types and methods
+	 *
+	 * Some of these are for reference and completeness only since
+	 * rxe does not currently support RD transport
+	 * most of this could be moved into Infiniband core. ib_pack.h has
+	 * part of this but is incomplete
+	 *
+	 * Header specific routines to insert/extract values to/from headers
+	 * the routines that are named __hhh_(set_)fff() take a pointer to a
+	 * hhh header and get(set) the fff field. The routines named
+	 * hhh_(set_)fff take a packet info struct and find the
+	 * header and field based on the opcode in the packet.
+	 * Conversion to/from network byte order from cpu order is also done.
+	 */
```

Comment:
Is this comment relevant to this layer? 🤔

Reply:
del